doc/dev/generatedocs.rst

```diff
@@ -42,12 +44,20 @@ Ubuntu 14.04:
 - ``ant``
 - ``cython``
 - ``librbd-dev``
+- ``zlib1g-dev``
 
 Execute ``apt-get install`` for each dependency that isn't installed
 on your host.::
 
-	sudo apt-get install python-dev python-pip python-virtualenv libxml2-dev libxslt-dev doxygen ditaa graphviz ant cython librbd-dev
+	sudo apt-get install gcc python-dev python-pip python-virtualenv libxml2-dev libxslt-dev doxygen ditaa graphviz ant cython librbd-dev zlib1g-dev
```

Comment:
@flaf i think what @ktdreyer suggests is to just:

* recommend the document writer or whoever wants to build the doc to use `./admin/build-doc` here. instead of maintaining the dependency list in two places. and this one is easier to grow stale over the time.

and i think we might want to update that script instead if preparing the building environment involves more dependencies.